### ISSUE TITLE
output of testing code

### ISSUE THREAD

**Issue by DecentMakeover** [XgTu/GFA-CNN#2]
Ho

Thanks for sharing your work, I am facing some difficulty understanding what is the output of testing code, it is printing steps and tccanti percentage. Also i wanted to know if there were any output images being stored.

Thanks in advance.

**Comment by XgTu** [comment]
@DecentMakeover 
Hi, the output for testing is 1/0 value for each image, representing if the images is classified as spoofing/live. The output images from FDA can be stored.

**Comment by DecentMakeover** [comment]
Hmmm, but when i run the testing code, it writes output to a text file with some accuracy metrics. Am I missing something here?

**Comment by XgTu** [comment]
@DecentMakeover 
Yes, I also compute the accuracy for all images that have been classified.

**Comment by DecentMakeover** [comment]
Okay let me check again-

On 25-Apr-2019, at 6:11 PM, Xg. Tu <notifications@github.com<mailto:notifications@github.com>> wrote:


@DecentMakeover<https://github.com/DecentMakeover>
Yes, I also compute the accuracy for all images that have been classified.

**Comment by jiagh2010** [comment]
你好，这个code分为两步，运行test_codes代码时发现，是要将图片resize至于
224*224，但是我看test images 都是320*240的？？代码里并没有做这步，这是要自己resize？？还是我运行的有偏差？？
